binance/client.py

```diff
@@ -870,7 +870,7 @@ def get_historical_klines(self, symbol, interval, start_str, end_str=None,
         :return: list of OHLCV values
 
         """
-        return self._historical_klines(symbol, interval, start_str, end_str=None, limit=500, spot=True)
+        return self._historical_klines(symbol, interval, start_str, end_str=end_str, limit=limit, spot=True)
```

Comment:
This is needed, however `get_historical_klines` does not allow for spot to be set either. (L855)

Reply:
Thanks! Do you mean it's supposed or not supposed to allow spot?
I just created #800 separately if you need spot option enabled.